sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -159,11 +192,31 @@ export default defineComponent({
           });
         }
       });
+      const applicationVersionHistory =
+        await ApplicationService.shared.getApplicationVersionHistory(
+          props.applicationId,
+        );
+      if (applicationVersionHistory.length) {
+        applicationVersionHistory.forEach((application) => {
+          applicationHistory.value.push({
+            title: `${application.submittedDate}`,
```

Comment:
I belive business confirmed that we can Prefix with `Submitted on`

![image](https://github.com/user-attachments/assets/e9a03d6a-e3a6-42f9-b023-3c23e50040e5)